inmaputil/wrapper.html

```diff
@@ -0,0 +1,27 @@
+{{ define "base" }}
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>InMap Web Interface</title>
```

Comment:
Maybe just InMAP, so people don't think this is coming to them over then internet?